wger/gym/migrations/0001_initial.py

```diff
@@ -20,8 +20,10 @@ class Migration(migrations.Migration):
                 ('timestamp_created', models.DateTimeField(auto_now_add=True)),
                 ('timestamp_edited', models.DateTimeField(auto_now=True)),
                 ('note', models.TextField(verbose_name='Note')),
-                ('member', models.ForeignKey(related_name='adminusernote_member', editable=False, to=settings.AUTH_USER_MODEL)),
-                ('user', models.ForeignKey(related_name='adminusernote_user', editable=False, to=settings.AUTH_USER_MODEL)),
+                ('member', models.ForeignKey(related_name='adminusernote_member', editable=False,
```

Comment:
Since migrations are auto generated, I'd say avoid making changes on them to avoid tampering with the logic.
http://flake8.pycqa.org/en/latest/user/configuration.html